packages/techdocs-cli/src/commands/generate/generate.ts

```diff
@@ -85,6 +86,7 @@ export default async function generate(cmd: Command) {
         }
       : {}),
     logger,
+    ...(process.env.LOG_LEVEL === "debug" ? { logStream: stdout } : {}),
```

Comment:
Sometimes errors from mkdocs are not output so this allows us to see the underlying issue.